CHANGELOG.md

```diff
@@ -3,6 +3,11 @@
 All notable changes to this project will be documented in this file.
 See [Conventional Commits](https://conventionalcommits.org) for commit guidelines.
 
+# 3.9.1 (2020-07-07)
```

Comment:
Shouldn't this be 4.0.1?

Reply:
Or maybe not... Should it go below 4.0? I almost feel like we need a different changelog for major versions if we're supporting both...